tools/c7n_org/c7n_org/cli.py

```diff
@@ -189,9 +188,10 @@ def init(config, use, debug, verbose, accounts, tags, policies, resource=None, p
     return accounts_config, custodian_config, executor
 
 
-def resolve_regions(regions):
+def resolve_regions(regions, account):
     if 'all' in regions:
-        client = boto3.client('ec2')
+        session = get_session(account, 'c7n-org', "us-east-1")
```

Comment:
could explain this signature change, we don't really want or need an account session, and that has implications wrt to needing to do sts role assumes just to resolve regions. is this because you have differential region enablement by account?